File: CrackDemo/CrackDemo/ViewController.m
//
//  ViewController.m
//  CrackDemo
//
//  Created by caijinzhu on 2017/11/6.
//  Copyright © 2017年 caijinzhu. All rights reserved.
//

#import "ViewController.h"

@interface ViewController()

@property (weak) IBOutlet NSTextField *tintLabel;

@property (weak) IBOutlet NSTextField *textField;

@end



@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    // Do any additional setup after loading the view.
}
- (IBAction)checkCode:(NSButton *)sender {
    if ([_textField.stringValue isEqualToString:@"1234"]) {
        _tintLabel.stringValue = @"验证通过";
    }else{
        _tintLabel.stringValue = @"验证失败";
    }
}


- (void)setRepresentedObject:(id)representedObject {
    [super setRepresentedObject:representedObject];

    // Update the view, if already loaded.
}


@end

File: RectDemo/RectDemo/ViewController.swift
//
//  ViewController.swift
//  RectDemo
//
//  Created by alexiuce  on 2017/9/30.
//  Copyright © 2017年 alexiuce . All rights reserved.
//

import Cocoa

class ViewController: NSViewController {

    @IBOutlet weak var label1: MYTextField!
    @IBOutlet weak var label2: MYTextField!
    @IBOutlet weak var label3: MYTextField!

    @IBOutlet weak var label4: MYTextField!
    
    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
    }

    override var representedObject: Any? {
        didSet {
        // Update the view, if already loaded.
        }
    }

    @IBAction func changeLabel2(_ sender: Any) {
        label2.stringValue = label2.stringValue == "" ? "a;kdjf;aiefjal;eijf;weojifadsfasdfsdffasdfdfa;efja;\n oweijfa;efija;woefija;dkfjaiefjai;efj" : ""
       
    }
    @IBAction func changeLabel3(_ sender: Any) {
       print("\(NSStringFromRect(label2.frame))")
    }
    @IBAction func resetDefault(_ sender: Any) {
    }
    
}

